src/components/ClickableStyle/ClickableStyle.tsx

```diff
@@ -78,7 +78,8 @@ const throwInvalidPropComboError = (variant: Variant, status: Status) => {
     return `'${validStatuses[0]}' and '${validStatuses[1]}'`;
   };
 
-  throw new Error(
+  // TODO: change to `throw new Error()` when invalid combos have been removed from downstream product
+  console.warn(
```

Comment:
Random idea... not sure if it's a good one or not...

Could add a `loose` prop that warns instead of throwing, so that the one instance can be less strict.

Reply:
Ooo interesting! It would be nice to make sure more invalid instances don't sneak in the meantime 🤔